src/audio/pcm_converter/pcm_converter.c

```diff
@@ -12,7 +12,14 @@
 
 #include <sof/audio/audio_stream.h>
 #include <sof/audio/pcm_converter.h>
-#include <sof/debug/panic.h>
+#include <sof/lib/uuid.h>
+#include <sof/trace/trace.h>
+
+/* bde59d40-0a1d-4205-b85a-9139f2bba316 */
+DECLARE_SOF_RT_UUID("pcm_conv", pcm_conv_uuid, 0xbde59d40, 0x0a1d, 0x4205,
+		    0xb8, 0x5a, 0x91, 0x39, 0xf2, 0xbb, 0xa3, 0x16);
+
+DECLARE_TR_CTX(pcm_conv_tr, SOF_UUID(pcm_conv_uuid), LOG_LEVEL_INFO);
 
```

Comment:
Wait. This change here doesn't look to be related to the commit message. Care to add it in a separate commit?

Reply:
I had the same initial impression but this seems related - looks required to add the pcm_conv_tr object needed in the trace

Reply:
Yes, it's needed for `tr_err` function. I added short sentence about it to commit message.